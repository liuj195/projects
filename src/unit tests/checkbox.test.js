import React from "react";
import CheckBox from "../components/buttons/checkbox";
import renderer from "react-test-renderer";

test("CheckBox renders correctly", () => {
  const fakeFunction = function() {};
  const component = renderer.create(
    <CheckBox
      onChange={fakeFunction}
      id={1}
      value="checkboxtest"
      name="checkboxtest"
      defaultChecked={true}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
