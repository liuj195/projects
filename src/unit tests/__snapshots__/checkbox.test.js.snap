// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CheckBox renders correctly 1`] = `
<li>
  <input
    id={1}
    name="checkboxtest"
    onChange={[Function]}
    type="checkbox"
    value="checkboxtest"
  />
  <label
    htmlFor={1}
  >
    checkboxtest
  </label>
  <br />
</li>
`;
